cypress/support/commands.js

```diff
@@ -49,3 +51,23 @@ Cypress.Commands.add(
             .closest('[data-test=dhis2-uicore-layer]')
             .click('topLeft')
 )
+
+/**
+ * Custom login command, can be used to login or switch between sessions.
+ * Will cache and restore cookies, localStorage, and sessionStorage. See:
+ * https://docs.cypress.io/api/commands/session
+ */
+Cypress.Commands.add('login', (user) => {
+    cy.session(
+        user,
+        () => {
+            cy.loginByApi(user)
+        },
+        {
+            validate: () => {
+                cy.validateUserIsLoggedIn(user)
+            },
+            cacheAcrossSpecs: false,
+        }
+    )
+})
```

Comment:
We could remove this, but I am still hopeful that at some point in the future logging in this way will actually work correctly. So it might be useful to keep this around.